lib/Dialect/TritonGPU/Transforms/Pipeliner/MatmulLoopPipeline.cpp

```diff
@@ -624,20 +753,11 @@ scheduleLoads(scf::ForOp forOp, tt::CoarseSchedule &schedule,
   }
   // Assign stages to the loads.
   for (auto [loadOp, indLevel, _] : loadOpToIndLevelAndUse) {
-    if (loadToInfo.count(loadOp) == 0)
+    if (loadsToPipeline.count(loadOp) == 0)
       continue;
     int stage = (maxIndirectionLevel - indLevel) * stagesBetweenLoads;
     schedule.insert(loadOp, stage, loadsClusters[indLevel]);
   }
-
-  // Distance from the load to the use.
```

Comment:
Why it was necessary to move it away from here to the level above?

Reply:
We are going to move this to the scheduling layer, so we should minimize the handshaking (i.e distToUse is used by lowering, so we should set it before lowering, instead of here during scheduling :]

Reply:
I missed that, makes perfect sense!